docker-compose.yaml

```diff
@@ -15,6 +15,8 @@ services:
       - ./mysql/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
   nextjs:
     container_name: next-auth-mysql-nextjs
+    environment:
```

Comment:
revert that changes. (not to make a conflict with devops) Also, usually we use .env file for that